librz/core/cmd/cmd_analysis.c

```
@@ -6528,3 +6350,100 @@ RZ_IPI RzCmdStatus rz_delete_global_imports_handler(RzCore *core, int argc, cons
 	rz_analysis_purge_imports(core->analysis);
 	return RZ_CMD_STATUS_OK;
 }
+
+RZ_IPI RzCmdStatus rz_analysis_data_handler(RzCore *core, int argc, const char **argv) {
+	int count = argc > 1 ? rz_num_math(core->num, argv[1]) : 2 + (core->blocksize / 4);
+	if (count < 1) {
+		count = 1;
+	}
```

better failing instead of silently changing things behind user's back.